Client/Frontend/Browser/Helpers/LaunchHelper.swift

```diff
@@ -29,14 +29,38 @@ public actor LaunchHelper {
     let task = Task {
       // Load cached data
       // This is done first because compileResources and loadCachedRuleLists need their results
-      async let loadBundledResources: Void = ContentBlockerManager.shared.loadBundledResources()
-      async let filterListCache: Void =  FilterListResourceDownloader.shared.loadCachedData()
-      async let adblockResourceCache: Void = AdblockResourceDownloader.shared.loadCachedData()
+      async let loadBundledResources: Void = measuredTask(
+        label: "bxx ContentBlockerManager.shared.loadBundledResources()", callback: {
```

Comment:
Can we make these labels more readable? (i.e. "Load Bundled Resources")

Reply:
I will remove this logging for now and will wait for you to get back to get this done right.